src/pages/index.astro

```diff
@@ -158,7 +170,21 @@ import MenuCard from '../components/MenuCard.astro';
     </div>
     <div>
       <h2>Stay in touch</h2>
```

Comment:
Weird to me that this is 'stay in touch' and below in the deleted class it says 'get in touch' - i would have expected the latter or for the title to be 'contact us'. Stay in touch sounds more like the mailing list, for which there is a big button above.

Reply:
Makes sense for email. The wording is from the design page but we should re-visit it.